src/server/migrations/20200930112056_add_notifications_table.js

```diff
@@ -1,23 +1,22 @@
-
 exports.up = function(knex) {
-    return knex.schema.createTable('notifications', (table) => {
-        table.increments();
-        table.integer('fk_user_id').unsigned();
-        table.foreign('fk_user_id').references('users.id');
-        table.string('message').notNullable();
-        table.boolean('has_been_read').notNullable();
-        table
-          .datetime('created_at')
-          .defaultTo(knex.fn.now())
-          .notNullable();
-        table
-          .datetime('updated_at')
-          .defaultTo(knex.fn.now())
-          .notNullable();
-        table.datetime('deleted_at');
-      });
+  return knex.schema.createTable('notifications', (table) => {
+    table.increments();
+    table.integer('fk_user_id').unsigned();
+    table.foreign('fk_user_id').references('users.id');
+    table.string('message').notNullable();
+    table.boolean('has_been_read').notNullable();
+    table
+      .datetime('created_at')
+      .defaultTo(knex.fn.now())
+      .notNullable();
+    table
+      .datetime('updated_at')
+      .defaultTo(knex.fn.now())
+      .notNullable();
+    table.datetime('deleted_at');
+  });
 };
 
 exports.down = function(knex) {
-    return knex.schema.dropTable('notifications');
+  return knex.schema.dropTable('notifications');
```

Comment:
All of this is not related to your branch. And you have not committed this as a seperate commit. Remember to check the files changed **everytime** you do a PR